test/run-drun/ok/reject-bug.drun-run.ok

```diff
@@ -1,4 +1,4 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-Ok: Reject: abcdefgh
 Ok: Reject: abcdefghi
+Ok: Reject: abcdefgh
```

Comment:
Does the order change because of undefined execution order of the fire-and-forget calls?

Reply:
**Update**: This is due to a race condition in `drun`. To be fixed by https://github.com/dfinity/ic/pull/552.